File: android/src/main/java/com/reactlibrary/RNAutoCompleteTextView.java
package com.reactlibrary;

import android.content.Context;
import android.content.res.Resources;
import android.graphics.Color;
import android.util.TypedValue;
import android.widget.LinearLayout.LayoutParams;

import androidx.annotation.RequiresApi;
import androidx.appcompat.widget.AppCompatAutoCompleteTextView;


@RequiresApi(21)
public final class RNAutoCompleteTextView extends AppCompatAutoCompleteTextView {
    int eventCount;
    int jsEventCount;
    public RNAutoCompleteTextView(Context context) {
        super(context);
        this.setInputType(524288);
        this.setTextColor(Color.parseColor("#333333"));
        this.setTextSize(14.0F);
        float value = 8;
        Resources resources = context.getResources();
        int dpValue = (int)TypedValue.applyDimension(1, value, resources.getDisplayMetrics());
        this.setPadding(0, this.getPaddingTop(), 0, dpValue);
        LayoutParams layoutParams = new LayoutParams(-1, -2);
        layoutParams.setMargins(0, 30, 0, 30);
        this.setLayoutParams((android.view.ViewGroup.LayoutParams)layoutParams);
        this.getBackground().setTint(Color.parseColor("#d3d3d3"));
        eventCount = 0;
        jsEventCount = 0;
    }

    public int getAndSetEventCount () {
        return eventCount++;
    }

    public int getEventCount () {
        return eventCount;
    }

    public int getJSEventCount () {
        return jsEventCount;
    }

    public void setJSEventCount (int count) {
        jsEventCount = count;
    }
}

File: android/src/main/java/com/reactlibrary/RNAutoCompleteTextView.kt
package com.rtsfinancial.carrierpro

import android.content.Context
import android.graphics.Color
import android.text.InputType
import android.util.TypedValue
import android.view.ViewGroup
import android.widget.LinearLayout
import androidx.appcompat.widget.AppCompatAutoCompleteTextView

class RNAutoCompleteTextView(context: Context) : AppCompatAutoCompleteTextView(context) {
    init {
        inputType = InputType.TYPE_TEXT_FLAG_NO_SUGGESTIONS
        setTextColor(Color.parseColor("#333333"))
        textSize = 14f
        val value = 8
        val dpValue = TypedValue.applyDimension(
                TypedValue.COMPLEX_UNIT_DIP,
                value.toFloat(),
                context.resources.displayMetrics).toInt()
        setPadding(0, paddingTop, 0, dpValue)
        val layoutParams = LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT)
        layoutParams.setMargins(0, 30, 0, 30)
        this.layoutParams = layoutParams
        background.setTint(Color.parseColor("#d3d3d3"))
    }
}